indico/modules/events/contributions/models/contributions.py

```diff
@@ -502,6 +502,15 @@ def allowed_types_for_editable(self):
         submitted_for = {editable.type.name for editable in self.editables}
         return [editable_type for editable_type in self.event.editable_types if editable_type not in submitted_for]
 
+    @property
+    def enabled_editable_types(self):
```

Comment:
Not sure about the naming... What about something like `enabled_editables` giving you directly the list of editables which are enabled?

Also a docstring to explain in more detail what this does

```
Return the active editables submitted for the contribution.

These are all the editables which have been submitted, but only if
their type is still enabled for the event.
```

(not 100% satisfied with that text either, but can't think of something better right now)

Reply:
Why not use the word "type" in there? This is supposed to return the names of all the editable types that have been submitted and are also enabled at the moment.

Reply:
Yeah, if you keep returning type names that's fine - I'd have returned the editables themselves instead (and wrote the docstring based on that suggestion)

Reply:
What's the advantage of returning the entire editable?

Reply:
Being able to use it directly in the template instead of this:

```
                    {% for editable in contribution.editables -%}
                        {% if editable.type.name in enabled_editable_types %}
```

PS: Jinja supports `{% for foo in bar if whatever %}` in case you decide to not change it.

Reply:
Maybe something like this for the docstring?
```
Return all the submitted editabled with enabled types.

These are all the editables which have been submitted and their type
is still enabled for the event.
```

Reply:
"Return all submitted editables for enabled types."

i think we don't even need the rest in this case since it's already clear :)